config/package_resource.go

```diff
@@ -101,8 +116,16 @@ func (p *packageResouce) validate(ctx context.Context) (*ManagedResources, error
 		if !packages.DpkgExists {
 			return nil, fmt.Errorf("cannot manage Deb package because dpkg does not exist on the system")
 		}
+		switch p.GetDesiredState() {
+		case agentendpointpb.ApplyConfigTask_OSPolicy_Resource_PackageResource_INSTALLED:
```

Comment:
Why is **Installed** not an applicable desired state for deb?
Similar comment for other package types in this file

Reply:
Installed is the only applicable desired state, any other state is not applicable. 
I reorganized the statement to be much more readable.